.github/workflows/lint.yml

```diff
@@ -0,0 +1,18 @@
+on: [push]
```

Comment:
Should lint.yml be part of release.yml and have build depend on linting? I think that would make more sense and prevent bad code from making it in.

Reply:
I'm not sure that it does matter to do it on release. You can just forbid users to merge PR if the linter step is failed

Reply:
My thought was that I'd not want a dev release to build if there are linter failures, which can only be done from the same pipeline if I'm not mistaken 🤔